runtime/vm/createramclass.cpp

```diff
@@ -2175,7 +2215,11 @@ internalCreateRAMClassDone(J9VMThread *vmThread, J9ClassLoader *classLoader, J9C
 			if (elementClass == NULL) {
 				alreadyLoadedClass = hashClassTableAt(classLoader, J9UTF8_DATA(className), J9UTF8_LENGTH(className));
 			} else {
-				alreadyLoadedClass = elementClass->arrayClass;
+				alreadyLoadedClass = createdArrayClass(elementClass
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
+				, state->valueTypeFlags
```

Comment:
The syntax alignment of `, state->valueTypeFlags` seems a bit off, maybe add another tab or two.